Modules/Package.swift

```diff
@@ -48,7 +48,7 @@ let package = Package(
         .package(url: "https://github.com/zendesk/support_sdk_ios", from: "8.0.3"),
         // We can't use wordpress-rs branches nor commits here. Only tags work.
         .package(url: "https://github.com/Automattic/wordpress-rs", revision: "alpha-20250127"),
-        .package(url: "https://github.com/wordpress-mobile/GutenbergKit", from: "0.0.5"),
+        .package(url: "https://github.com/wordpress-mobile/GutenbergKit", branch: "task/make-title-optional"),
```

Comment:
Friendly reminder to update this to the merge commit in GutenbergKit's `trunk` branch before merging this work.